tests/playwright/utils/auth.ts

```diff
@@ -0,0 +1,320 @@
+import {
+  CognitoUserPool,
+  CognitoUser,
+  AuthenticationDetails,
+  CognitoUserSession,
+  IAuthenticationCallback,
+} from "amazon-cognito-identity-js";
+import { Page, expect } from "@playwright/test";
+import AWS from "aws-sdk";
```

Comment:
https://www.npmjs.com/package/aws-sdk

aws-sdk is in maintenance mode.  The current SDK is modular like:
https://www.npmjs.com/package/@aws-sdk/client-cloudformation